pom.xml

```diff
@@ -1741,6 +1768,9 @@
                         <!-- our tests are very verbose, let's keep the volume down -->
                         <redirectTestOutputToFile>true</redirectTestOutputToFile>
                         <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
+                        <excludes>
```

Comment:
Please include some comment about why we are doing this. I think it's about removing a default exclusion for static inner classes, but it is not obvious.